infra/blueprint-test/pkg/tft/terraform_test.go

```diff
@@ -242,9 +254,17 @@ func TestSetupOverrideList(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			emptyDir := newTestDir(t, "empty*", "")
+			t.Cleanup(func() {
+				if err := os.RemoveAll(emptyDir); err != nil {
+					t.Logf("Error removing temp dir %s: %v", emptyDir, err)
+				}
+			})
 			setupDir := newTestDir(t, "setup-*", tt.tfOutputs)
-			defer os.RemoveAll(emptyDir)
```

Comment:
same as above

Reply:
It's moved to the following `if` statement: `if err := os.RemoveAll(setupDir); err != nil {`